charmhelpers/fetch/ubuntu.py

```diff
@@ -242,8 +242,20 @@ def apt_cache(*_, **__):
     return ubuntu_apt_pkg.Cache()
 
 
-def apt_install(packages, options=None, fatal=False):
-    """Install one or more packages."""
+def apt_install(packages, options=None, fatal=False, override_env=None):
+    """Install one or more packages.
+
+    :param packages: Package(s) to install
+    :type packages: Option(str, List)
```

Comment:
The type here is (I think): `Option[str, List[str]]`

I'm not really sure why mypy went for square brackets `[]` vs parenthesis `()`, but sadly, and unintuitively, they did.  See http://mypy-lang.org/ for more details.

Reply:
:+1: